.editorconfig

```diff
@@ -1,11 +1,12 @@
 root = true
 
 [*]
-indent_style = space
+indent_style = tab
```

Comment:
Crap... there's gonna be a debate. I confess I'm more `space` than `tab`.

Reply:
I am too (and with 2 spaces) but a lot of code has been written with tabs ^^